_episodes/03-working-with-files.md

```diff
@@ -258,33 +258,33 @@ first or last `n` lines of a file. To print the first/last line of the
 file use:
 
 ~~~
-head -n 1 SRR098026.fastq
-tail -n 1 SRR098026.fastq
+$ head -n 1 SRR098026.fastq
+$ tail -n 1 SRR098026.fastq
 ~~~
 {: .bash}
 
 ## Creating, moving, copying, and removing
 
-Now we can move around in the file structure, look at files, search files,
-redirect. But what if we want to do normal things like copy files or move
-them around or get rid of them. Sure we could do most of these things
-without the command line, but what fun would that be?! Besides it's often
-faster to do it at the command line, or you'll be on a remote server
-like Amazon where you won't have another option.
+Now we can move around in the file structure, look at files, search files, and
+redirect output to new files or to other commands. But what if we want to copy files or move
+them around or get rid of them. Most of the time, you can do these sorts of file manipulations without the command line,
+but there will be come cases (like when you're working with a remote computer like we are for this lesson) where it will be
```

Comment:
come -> some?

Reply:
Like @swang8 I guess it should be 'some'